rest/route/project_routes.go

```diff
@@ -128,3 +131,82 @@ func projectPaginator(key string, limit int, args interface{}, sc data.Connector
 
 	return models, pages, nil
 }
+
+type versionsGetHandler struct {
```

Comment:
Tests for this?

Reply:
I had attempted to write tests initially, but mux makes mocking requests impossible so I would need to start a web server in order to test this. Since the test would just test some simple parameter validation and the core logic is already unit tested, I didn't write one